### PR TITLE
feat(Interaction): lever attach to game object - resolves #611

### LINKED ISSUE #611: VRTK_Lever connectedTo property for attaching to moving objects
@10FingerArmy The VRTK_Button script has a public member GameObject 'connectedTo' for attaching the button object to another game object (for movement purposes). The VRTK_Lever script does not. I added this locally for me in the same style as the button script. I think it would be a nice enhancement. Let me know if there is something issue I'm not seeing that would break it.

### PR DESCRIPTION
**Pull request by mlabus93**
Lever attaches to game object so that it is moveable with connected game
object. Previously, lever is unable to attach to game object and cannot
move without breaking.

### CODE REVIEW

**Comment by mlabus93** [comment]
@10FingerArmy Here is the pull request for the issue (#611) I had opened on adding the connectedTo property for the VRTK_Lever

**Comment by thestonefox** [comment]
@mlabus93 looks good to me
I'll see what @10FingerArmy thinks before merging

File: Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs [hunk 1]
```diff
@@ -19,6 +19,8 @@ public enum LeverDirection
             x, y, z
         }
 
+        [Tooltip("An optional game object to which the button will be connected. If the game object moves the button will follow along.")]
```
Comment (10FingerArmy) [inline Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs]: tooltip mentions button, should be lever

Comment (thestonefox) [inline Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs]: good catch!

Comment (mlabus93) [inline Assets/VRTK/Scripts/Controls/3D/VRTK_Lever.cs]: Whoops, that's why you shouldn't copy/paste code haha. I will amend and push again today, thanks.

**Comment by mlabus93** [comment]
@thestonefox @10FingerArmy The tooltip has been updated from button to lever. It's ready to merge now I think, if you guys see fit.

**Comment by mlabus93** [comment]
Woohoo! My first contribution! I'd like to start off by thanking my sponsors and I'd like to thank my wife and kids that don't exist yet and to all of my fans who drove me to make this possible.

**Pull request merged**